example/docker-compose/shared/tempo.yaml

```diff
@@ -1,4 +1,3 @@
-multitenancy_enabled: true
```

Comment:
did you check to see if all examples worked with this change ? particularly concerned about the otelcol multitenant one.

Reply:
I checked a couple, including that one. It configures multi-tenancy via a flag https://github.com/grafana/tempo/blob/main/example/docker-compose/otel-collector-multitenant/docker-compose.yaml#L23

Reply:
+1